package.json

```diff
@@ -25,6 +25,7 @@
     "build_netlify": "NODE_ENV=production npm run disallow_robots_txt && npm run production && npm run fetch_blogs && npm run fetch_projects"
   },
   "dependencies": {
-    "eclipsefdn-hugo-solstice-theme": "0.0.58"
+    "eclipsefdn-hugo-solstice-theme": "0.0.63",
+    "jquery-eclipsefdn-api": "0.0.29"
```

Comment:
I will create a release of hugo to include "jquery-eclipsefdn-api": "0.0.29". Let's remove it!